CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 ## 0.10.0 (February 7, 2020)
 * Enhancements
     * Changed the default help text to make `help -v` more discoverable
-    * **set** command now supports tab-completion of values
+    * **set** command now supports tab completion of values
```

Comment:
So are we now standardizing on "tab completion" instead of "tab-completion"?  Works for me.